pkg/etcdmgr/bootstrap/discovery.go

```diff
@@ -18,49 +18,71 @@ package bootstrap
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/coreos/etcd/client"
 	"github.com/coreos/etcd/store"
 	"golang.org/x/net/context"
 	"golang.org/x/net/context/ctxhttp"
 )
 
-func isQuorumFull(token string) (bool, error, []string) {
```

Comment:
omg thank you for reordering the return params so error is last, as it should be in idiomatic go